lib/facter/swap_file_size.rb

```diff
@@ -0,0 +1,19 @@
+Facter.add('swap_file_size') do
```

Comment:
Is this even possible? How can the fact name be dynamic in this way? I realize there could be multiple swap files, however the correct approach would be to make `$::swap_file_size` return a hash {"/swapfile" => size, "/swapfile2" => size}, however this requires facter `stringifyfacts` to be disabled which is set to `true` in our current puppet deployment:
https://docs.puppet.com/puppet/3.8/reference/configuration.html#stringifyfacts

I think this is a good next step for `$::swap_file_size` but is not realistic to implement currently.